shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/Platform.kt

```diff
@@ -44,5 +44,12 @@ class PlatformManager(distribution: Distribution = Distribution(), experimental:
     val hostPlatform = platforms.getValue(host)
 
     fun loader(target: KonanTarget) = loaders.getValue(target)
+
+    /**
+     * TODO: Don't forget to delete this field and replace all its usages to `enabled`.
+     */
+    val filteredOutEnabledButNotSupported
```

Comment:
Maybe better just `supported`?

Reply:
it's supposed to be removed as soon as platforms will be available.

Reply:
This may be needed as more permanent mechanism, but we can do that separately.